notebooks/notebooks/iclr2022/estimate-flops-katago.ipynb

```diff
@@ -0,0 +1,662 @@
+{
```

Comment:
<p>Bit sad that it won't run in Docker, makes it a bit harder to reproduce. How hard would it be to get Docker to work? Naively, could this be as simple as swapping <code>FROM python:3.10-slim</code> to <code>FROM nvidia/cuda:11.3.0-devel-ubuntu20.04</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Updated our docker config and now our notebook works!